master/getting-started/kubernetes/host-endpoints.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
Yep I was chatting with @fasaxc earlier and that was part of one of his suggestions to add a hep example.  I'll also add a note that this only applies to etcd (non-KDD) and must be configured with calicoctl.